G/GDAL/build_tarballs.jl

```diff
@@ -3,24 +3,24 @@
 using BinaryBuilder, Pkg
 
 name = "GDAL"
-upstream_version = v"3.10.3"
+upstream_version = v"3.11.0"
 # The version offset is used for two purposes:
 # - If we need to release multiple jll packages for the same GDAL
 #   library (usually for weird packaging reasons) then we increase the
 #   offset because we usually cannot release the same version twice.
 # - Minor versions of GDAL are usually binary incompatible because
 #   they increase the shared library soname. To encode this, we
 #   increase the major version number of the version offset.
-version_offset = v"2.0.0"
+version_offset = v"3.0.0"
```

Comment:
Is this a breaking change?

Reply:
Yes. The release notes explicitly say so, citing an increase in the shared library major version number as reason.